### ISSUE TITLE
docker-compose.yml file?

### ISSUE THREAD

**Issue by eric-chou** [omoponfhir/omoponfhir-main#1]
Hello,
I am a researcher with the Dept of Biomedical Informatics at the University of Pittsburgh and we found this project via omoponfhir.org while we were investigating ways to adapt our own OMOP data with FHIR. 

While looking to set this up locally via the directions in the README, it looks like I was able to get the smart-postgresql dependency set up, but under the "[How to install and run](https://github.com/omoponfhir/omoponfhir-main#how-to-install-and-run)" step for this project, I have not been able to start the application without a docker-compose.yml file. Is there supposed to be a docker-compose.yml file included in this top directory alongside the Dockerfile? I am not very experienced with Docker so I am having trouble with how to proceed.

Thank you very much for your time!
-Eric

**Comment by myungchoi** [comment]
Eric, you are right. I missed the docker-compose file when I am moving to the new OMOPonFHIR organization.  I also updated the README file. The contents are for previous repo.

Please let us know here if you run into an issue installing and running the server. 

Please note that OMOPonFHIR-main repo contains submodules, which you need to add --recurse in your git clone command.

**Comment by eric-chou** [comment]
It works great now thank you for your help! One thing I had to tweak for me was in the pom.xml under "omoponfhir-stu-server" I had to remove the "javax.persistence" dependency to get it to work. My project was throwing an error in which it saw that as a duplicate dependency, as per the error in the screenshot below. Perhaps this may have been unique to my local machine, but just letting you know as a heads up just in case.
![image](https://user-images.githubusercontent.com/9423731/62069152-a25b8c00-b205-11e9-811e-b469838c2066.png)

Thanks!
